src/tool/xmeta/cpp/models/formal_parameter_model.h

```diff
@@ -25,6 +25,12 @@ namespace xlang::xmeta
             m_type{ std::move(type) }
         { }
 
+        formal_parameter_model(std::string_view const& id, parameter_semantics sem, type_ref&& type) :
```

Comment:
nit picky, but are there other types of parameter models OTHER than formal? Shouldn't this just be parameter_model?

Reply:
I will change this in a later PR